### ISSUE TITLE
Add new cell type term 

### ISSUE THREAD

**Issue by pnejad** [HumanCellAtlas/ontology#46]
For new term requests, please provide the following information:

## Preferred term label

cortical nephrogenic zone cells

## Textual definition

Cells from the cortical nephrogenic zone.

## Suggested parent term

kidney cortical cell
https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0002681

## Use case

This is a cell type for one of the GEO datasets we are wrangling.

**Comment by zoependlington** [comment]
One for @dosumis for CL.

**Comment by dosumis** [comment]
Note to self - relevant Uberon term: http://www.ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/UBERON_0009871

**Comment by paolaroncaglia** [comment]
@dosumis 
Following up on this ticket. I'd like to help but wanted to make sure if you had already worked on this, as you noted the Uberon term you wanted to use as part_of. I can't see any ticket in the CL tracker for 'cortical nephrogenic zone cell', and I believe that the existing CL term 'nephrogenic mesenchyme stem cell' is not an exact match for Parisa's request. Could you please confirm when you're back from leave? Thanks.

**Comment by paolaroncaglia** [comment]
Hi @pnejad ,
Could you please provide details of the GEO dataset you requested this term for?
If there's a publication associated with it, it'll be useful as I suspect that the "cortical nephrogenic zone" you refer to is indeed the same as http://purl.obolibrary.org/obo/UBERON_0009871 'nephrogenic zone'. In fact, that is defined as the outer portion of the renal _cortex_, and I suspect that the "cortical" in "cortical nephrogenic zone" is redundant (or an exact synonym). 
I will then add the cell type term in CL (as discussed with David today).
Thanks,
Paola

**Comment by pnejad** [comment]
@paolaroncaglia This ticket was made for GSE102596.

https://pubmed.ncbi.nlm.nih.gov/29449449/
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102596

**Comment by paolaroncaglia** [comment]
@pnejad 
I opened a [CL ticket](https://github.com/obophenotype/cell-ontology/issues/824) to create the new term for cortical nephrogenic zone cell (note that the primary label will be 'nephrogenic zone cell', but "cortical nephrogenic zone cell" will stay as an exact synonym). I also opened a [Uberon ticket](https://github.com/obophenotype/uberon/issues/1692) to add "cortical nephrogenic zone" and "cortical nephrogenic niche" to UBERON_0009871 'nephrogenic zone'. I'll close this HCAO ticket now. Thanks.
